test/frontend/reducers/test_concept_systems.js

```diff
@@ -1,12 +1,14 @@
 import { expect } from '../test_helper';
-import conceptSystems  from '../../../webpack/reducers/concept_systems_reducer';
-import _ from 'lodash';
+
+import { byIdReducer } from '../../../webpack/reducers/reducer_generator';
```

Comment:
Is this file in here as a bug fix or something, seems unrelated.  Not a blocker just curious